docs_guidelines/how_to/create_lists.rst

```diff
@@ -0,0 +1,113 @@
+============
+Create lists
+============
+
+This guide shows how to create a list in your documentation.
+
+Pre-requisites
+==============
+
+This guide assumes your Qiskit project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+`sphinx-quickstart <https://www.sphinx-doc.org/en/master/man/sphinx-quickstart.html>`_.
```

Comment:
I think pre-requisites here is not necessary because this is a pure `RST` guide. But it's good that you have thought about that.

Reply:
My reasoning was that in order for all this `RST` to do anything in your docs you needed to use Sphinx, so you needed the docs Sphinx project. I might be wrong of course!

Reply:
It's true but it's a bit redundant in this case because that's required for all the how-to guides here, while some other functionalities explicitly requires sphinx.